containers/unit-tests/build

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -ex
+
+dir=$(dirname "$0")
+
+sudo docker build --build-arg debian_arch=amd64 --build-arg personality=linux64 -t cockpit/unit-tests ${dir}
```

Comment:
Let's please not put sudo into the script, and instead document that it needs to be called through sudo. Otherwise calling the script can hang forever asking for a password on the second or third command, which is unhelpful for a "start and then move your attention somewhere else" workflow.

Plus, let's [support podman](https://github.com/cockpit-project/cockpituous/blob/master/Makefile#L12) as well, which can do the builds without root privs even. That part would be fine as a follow-up, but it's also a trivial update.

Reply:
I kept the `sudo` (despite similar reservations) because of the same compatibility reason that I kept the make targets for.  I'm happy to get rid of it :)